projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -118,6 +118,14 @@ public function __construct( $attributes, $content = null ) {
 			$default_to      .= $post_author->user_email;
 		}
 
+		if ( ! empty( self::$forms ) ) {
+			// Ensure 'id' exists in $attributes before trying to modify it
+			if ( ! isset( $attributes['id'] ) ) {
+				$attributes['id'] = '';
+			}
+			$attributes['id'] = $attributes['id'] . '-' . ( count( self::$forms ) + 1 );
```

Comment:
Fine as is, but noting that [`wp_unique_id()`](https://developer.wordpress.org/reference/functions/wp_unique_id/) can be handy here.